tests/virt/conftest.py

```
@@ -237,7 +240,28 @@ def hco_cr_with_mdev_permitted_hostdevices_scope_session(hyperconverged_resource
+    """
+    with ResourceEditorValidateHCOReconcile(
+        patches={hyperconverged_resource_scope_session: {"spec": {"featureGates": {DISABLE_MDEV_CONFIGURATION: True}}}},
+        list_resource_reconcile=[KubeVirt],
+        wait_for_reconcile_post_update=True,
+    ):
+        yield
+
+
+@pytest.fixture(scope="session")
+def gpu_nodes_labeled_with_vgpu_config(
+    hco_with_disable_mdev_configuration,
```

Ruff (0.15.13)

[warning] Unused function argument: `hco_with_disable_mdev_configuration`

(ARG001)